sys/kern/src/arch/rv64/stimer.rs

```diff
@@ -0,0 +1,12 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#[no_mangle]
+pub unsafe fn set_timer(tick_divisor: u32) {
+    // TODO: SBI Call
```

Comment:
Can we just use `stime`,  Are we supporting Sstc?

Reply:
For SSTC-carrying devices, yes. I believe we'll have to implement both for potential upstream acceptance.

Reply:
This is now blocked waiting on https://github.com/rivosinc/riscv/pull/4